src/utils/optimizely.js

```diff
@@ -0,0 +1,65 @@
+export const EVENTS = {
+  ENROLLMENT_CLICK: 'enterprise_learner_portal_enrollment_click',
+}
+
+export const EXPERIMENT_0 = {
+  ID: 21468970242,
+  VARIANT_IDS: {
+    CONTROL: 21446530045,
+    VARIANT_1: 21458980337,
+  },
+}
+
+export const getOptimizelyProps = () => ({
+  src: process.env.ENVIRONMENT === 'production'
+    ? `${process.env.BASE_URL}/optimizelyjs/${process.env.OPTIMIZELY_PROJECT_ID}.js`
+    : `https://cdn.optimizely.com/js/${process.env.OPTIMIZELY_PROJECT_ID}.js`,
+  rel: 'preload',
+});
+
+export const getActiveExperiments = () => {
+  window['optimizely'] = window['optimizely'] || [];
+  return window['optimizely'].get('state').getActiveExperimentIds().map(Number);
```

Comment:
[curious] If `window.optimizely` gets defaulted to `[]` (from `getActiveExperiments` above), will `.get()` be a valid function call?

<img width="332" alt="image" src="https://user-images.githubusercontent.com/2828721/165399343-d493289d-37b5-40ad-9ba8-302d33ec3a83.png">

Reply:
Yeah, I don't know if I like this pattern either (again, copied from examples in docs).  Maybe we can switch all these functions to doing a check/early-return like this?
```js
if (!!window['optimizely']) {return}
window['optimizely'].push(...)
```